aspnetcore/blazor/tooling.md

```diff
@@ -465,7 +473,7 @@ A *solution* is a container to organize one or more related code projects. [Visu
 Tooling outside of Visual Studio can interact with solution files:
 
 * The [.NET CLI](/dotnet/core/tools/) can create solution files and list/modify the projects in solution files via the [`dotnet sln` command](/dotnet/core/tools/dotnet-sln). Other .NET CLI commands use the path of the solution file for various publishing, testing, and packaging commands.
-* [Visual Studio Code](https://code.visualstudio.com) can execute the `dotnet sln` command and other .NET CLI commands through its integrated terminal but doesn't use the settings in a solution file directly.
+* [Visual Studio Code](https://code.visualstudio.com) can execute the [`dotnet sln` command](/dotnet/core/tools/dotnet-sln) and other [.NET CLI commands](/dotnet/core/tools/) through its integrated **Terminal** that interact with the solution file's contents, but VS Code doesn't use the settings in a solution file directly.
```

Comment:
Hmm, is this correct? I just tried out adding a project to an existing .NET solution using the Solution Explorer in C# Dev Kit and it updated the solution file accordingly.

Reply:
Yes, it's stale content now. I'll remove this remark.

```suggestion
```